tajo-core/src/main/java/org/apache/tajo/engine/function/window/Rank.java

```diff
@@ -48,7 +48,8 @@ public Rank() {
 
   public static boolean checkIfDistinctValue(RankContext context, Tuple params) {
     for (int i = 0; i < context.latest.length; i++) {
-      if (!context.latest[i].equalsTo(params.asDatum(i)).isTrue()) {
+      if ((context.latest[i].isNotNull() || params.asDatum(i).isNotNull())
```

Comment:
Because, in rank(), null values are regarded as the same unlike our other implementations. Please refer to https://en.wikipedia.org/wiki/Three-valued_logic.